lib/personalAccessKey.ts

```diff
@@ -132,6 +132,25 @@ async function getNewAccessToken(
   return accessTokenResponse;
 }
 
+async function getNewAccessTokenByAccountId(
+  accountId: number
+): Promise<AccessToken> {
+  const account = getAccountConfig(accountId) as PersonalAccessKeyAccount;
+  if (!account) {
+    throwErrorWithMessage(`${i18nKey}.errors.accountNotFound`, { accountId });
+  }
+  const { auth, personalAccessKey, env } = account;
+  const authTokenInfo = auth && auth.tokenInfo;
```

Comment:
I think it'd be good to wrap this in a try...catch block, in case `getNewAccessToken` fails. Also, what happens if `auth.tokenInfo` is falsey? Then `authTokenInfo` will be `null`--will `getNewAccessToken` accept `null`?

Reply:
simplified it to a bunch of coalesces, it's fine if expiresAt is not provided. 

I'm not sure the benefits of grouping this in a try catch is, could you elaborate?

Reply:
@wung-hubspot, just saw this. I looked at `getNewAccessToken`, where there is a try catch block, so ignore this comment. Going to approve this now.